lib/debug/server_dap.rb

```diff
@@ -64,6 +65,14 @@ def self.setup debug_port
       end
     end
 
+    def self.register_request *names, &b
```

Comment:
The style/naming is not what I'd typically use, but I've tried keep it consistent with what's already in use.